src/stories/Token.stories.tsx

```diff
@@ -0,0 +1,238 @@
+      )
+    }
+  ]
+} as Meta
+
+const SingleExampleContainer: React.FC<{label?: string}> = ({children, label}) => (
+  <Box
+    display="flex"
+    sx={{
+      alignItems: 'start',
+      flexDirection: 'column',
+      gap: get('space.0')
+    }}
+  >
+    {label ? (
+      <Text fontSize={0} color="text.tertiary">
+        {label}
+      </Text>
+    ) : null}
+    {children}
+  </Box>
+)
+
+const ExampleCollectionContainer: React.FC = ({children}) => (
+  <Box
+    display="flex"
+    sx={{
+      alignItems: 'start',
+      flexDirection: 'column',
+      gap: get('space.6')
+    }}
+  >
+    {children}
+  </Box>
+)
+
+export const defaultToken = () => (
+  <ExampleCollectionContainer>
+    <SingleExampleContainer label="Resting">
+      <Token text="Default Token" />
+    </SingleExampleContainer>
+    <SingleExampleContainer label="Interactive">
+      <Box
+        display="flex"
+        sx={{
+          alignItems: 'start',
+          gap: get('space.2')
+        }}
+      >
+        <Token as="a" href="http://google.com/" text="Link" />
+        <Token as="button" onClick={() => console.log('clicked')} text="Button" />
+        <Token as="span" tabIndex={0} onFocus={() => console.log('focused')} text="Focusable Span" />
+      </Box>
+    </SingleExampleContainer>
+    <SingleExampleContainer label="w/ onRemove passed">
+      <Token
+        text="Default Token"
+        onRemove={() => {
+          console.log('remove me')
+        }}
+      />
+    </SingleExampleContainer>
+    <SingleExampleContainer label="w/ onRemove passed and the token is clickable">
+      <Box
+        display="flex"
+        sx={{
+          alignItems: 'start',
+          gap: get('space.2')
+        }}
+      >
+        <Token
+          as="a"
+          href="http://google.com/"
+          text="Link"
+          onRemove={() => {
+            console.log('remove me')
+          }}
+        />
+        <Token
+          as="button"
+          onClick={() => console.log('clicked')}
+          text="Button"
+          onRemove={() => {
+            console.log('remove me')
+          }}
+        />
+        <Token
+          as="span"
+          tabIndex={0}
+          onFocus={() => console.log('focused')}
+          text="Focusable Span"
+          onRemove={() => {
+            console.log('remove me')
+          }}
+        />
+      </Box>
+    </SingleExampleContainer>
+    <SingleExampleContainer label="w/ leadingVisual passed">
+      <Token text="Default Token" leadingVisual={() => <VerifiedIcon size={12} />} />
+    </SingleExampleContainer>
+    <SingleExampleContainer label="isSelected">
+      <Token text="Default Token" isSelected={true} />
+    </SingleExampleContainer>
+  </ExampleCollectionContainer>
+)
+
+export const profileToken = () => (
+  <ExampleCollectionContainer>
+    <SingleExampleContainer label="Resting">
+      <TokenProfile size="lg" avatarSrc="https://avatars.githubusercontent.com/mperrotti" text="Mike Perrotti" />
```

Comment:
Should we use children instead of text here? 

```jsx
<TokenProfile size="lg" avatarSrc="https://avatars.githubusercontent.com/mperrotti">
   Mike Perrotti
</TokenProfile>
```

Reply:
When I use `children`, I expect to be able to pass a `ReactNode`, so it feels a little strange to limit it to `string`. Also, we use `text` instead of `children` in the [ActionList.Item component](https://github.com/primer/react/blob/9f39518224973dbc33dcc697fef1877ab2c4510c/src/ActionList/Item.tsx#L57).

This could just be a personal preference though, so I'm open to using `children` if other reviewers feel like this makes more sense.

Reply:
I'm okay with either, children feels like the way to put contents inside that already exists.

> When I use children, I expect to be able to pass a ReactNode, so it feels a little strange to limit it to string. 

That makes sense! I guess it's contextual. In the case of a label, you would pass a string in children. The type check is to warn us if it is stretched/misused beyond that.

> Also, we use text instead of children in the ActionList.Item component.

That's another one I'd like to change, proposal coming soon 😅

Reply:
> That's another one I'd like to change, proposal coming soon 😅

Good to know!